infra/tpu-ci/modules/arc-v4-container-cluster/main.tf

```diff
@@ -0,0 +1,97 @@
+provider "google" {
```

Comment:
Can you move this module under `infra/terraform_modules`?

Reply:
Moved.